<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!-- NewPage -->
<html lang="en">
<head>
<!-- Generated by javadoc (version 1.7.0_17) on Mon Feb 17 10:24:31 CET 2014 -->
<title>NodeRetrieverTitan</title>
<meta name="date" content="2014-02-17">
<link rel="stylesheet" type="text/css" href="../../../../../../stylesheet.css" title="Style">
</head>
<body>
<script type="text/javascript"><!--
    if (location.href.indexOf('is-external=true') == -1) {
        parent.document.title="NodeRetrieverTitan";
    }
//-->
</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
<!-- ========= START OF TOP NAVBAR ======= -->
<div class="topNav"><a name="navbar_top">
<!--   -->
</a><a href="#skip-navbar_top" title="Skip navigation links"></a><a name="navbar_top_firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="../../../../../../overview-summary.html">Overview</a></li>
<li><a href="package-summary.html">Package</a></li>
<li class="navBarCell1Rev">Class</li>
<li><a href="class-use/NodeRetrieverTitan.html">Use</a></li>
<li><a href="package-tree.html">Tree</a></li>
<li><a href="../../../../../../deprecated-list.html">Deprecated</a></li>
<li><a href="../../../../../../index-files/index-1.html">Index</a></li>
<li><a href="../../../../../../help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList">
<li><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/Bio4jManager.html" title="class in com.ohnosequences.bio4j.titan.model.util"><span class="strong">Prev Class</span></a></li>
<li>Next Class</li>
</ul>
<ul class="navList">
<li><a href="../../../../../../index.html?com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html" target="_top">Frames</a></li>
<li><a href="NodeRetrieverTitan.html" target="_top">No Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_top">
<li><a href="../../../../../../allclasses-noframe.html">All Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_top");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<div>
<ul class="subNavList">
<li>Summary:&nbsp;</li>
<li>Nested&nbsp;|&nbsp;</li>
<li><a href="#field_summary">Field</a>&nbsp;|&nbsp;</li>
<li><a href="#constructor_summary">Constr</a>&nbsp;|&nbsp;</li>
<li><a href="#method_summary">Method</a></li>
</ul>
<ul class="subNavList">
<li>Detail:&nbsp;</li>
<li><a href="#field_detail">Field</a>&nbsp;|&nbsp;</li>
<li><a href="#constructor_detail">Constr</a>&nbsp;|&nbsp;</li>
<li><a href="#method_detail">Method</a></li>
</ul>
</div>
<a name="skip-navbar_top">
<!--   -->
</a></div>
<!-- ========= END OF TOP NAVBAR ========= -->
<!-- ======== START OF CLASS DATA ======== -->
<div class="header">
<div class="subTitle">com.ohnosequences.bio4j.titan.model.util</div>
<h2 title="Class NodeRetrieverTitan" class="title">Class NodeRetrieverTitan</h2>
</div>
<div class="contentContainer">
<ul class="inheritance">
<li>java.lang.Object</li>
<li>
<ul class="inheritance">
<li>com.ohnosequences.bio4j.titan.model.util.NodeRetrieverTitan</li>
</ul>
</li>
</ul>
<div class="description">
<ul class="blockList">
<li class="blockList">
<dl>
<dt>All Implemented Interfaces:</dt>
<dd>com.ohnosequences.bio4j.model.util.NodeRetriever</dd>
</dl>
<hr>
<br>
<pre>public class <span class="strong">NodeRetrieverTitan</span>
extends java.lang.Object
implements com.ohnosequences.bio4j.model.util.NodeRetriever</pre>
<dl><dt><span class="strong">Author:</span></dt>
  <dd>Pablo Pareja Tobes <ppareja@era7.com></dd></dl>
</li>
</ul>
</div>
<div class="summary">
<ul class="blockList">
<li class="blockList">
<!-- =========== FIELD SUMMARY =========== -->
<ul class="blockList">
<li class="blockList"><a name="field_summary">
<!--   -->
</a>
<h3>Field Summary</h3>
<table class="overviewSummary" border="0" cellpadding="3" cellspacing="0" summary="Field Summary table, listing fields, and an explanation">
<caption><span>Fields</span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Field and Description</th>
</tr>
<tr class="altColor">
<td class="colFirst"><code>protected <a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/Bio4jManager.html" title="class in com.ohnosequences.bio4j.titan.model.util">Bio4jManager</a></code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#manager">manager</a></strong></code>&nbsp;</td>
</tr>
</table>
</li>
</ul>
<!-- ======== CONSTRUCTOR SUMMARY ======== -->
<ul class="blockList">
<li class="blockList"><a name="constructor_summary">
<!--   -->
</a>
<h3>Constructor Summary</h3>
<table class="overviewSummary" border="0" cellpadding="3" cellspacing="0" summary="Constructor Summary table, listing constructors, and an explanation">
<caption><span>Constructors</span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colOne" scope="col">Constructor and Description</th>
</tr>
<tr class="altColor">
<td class="colOne"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#NodeRetrieverTitan(com.ohnosequences.bio4j.titan.model.util.Bio4jManager)">NodeRetrieverTitan</a></strong>(<a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/Bio4jManager.html" title="class in com.ohnosequences.bio4j.titan.model.util">Bio4jManager</a>&nbsp;bio4jManager)</code>&nbsp;</td>
</tr>
</table>
</li>
</ul>
<!-- ========== METHOD SUMMARY =========== -->
<ul class="blockList">
<li class="blockList"><a name="method_summary">
<!--   -->
</a>
<h3>Method Summary</h3>
<table class="overviewSummary" border="0" cellpadding="3" cellspacing="0" summary="Method Summary table, listing methods, and an explanation">
<caption><span>Methods</span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tr class="altColor">
<td class="colFirst"><code>com.ohnosequences.bio4j.blueprints.model.nodes.AlternativeProductNode</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getAlternativeProductInitiationNode()">getAlternativeProductInitiationNode</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>com.ohnosequences.bio4j.blueprints.model.nodes.AlternativeProductNode</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getAlternativeProductPromoterNode()">getAlternativeProductPromoterNode</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>com.ohnosequences.bio4j.blueprints.model.nodes.AlternativeProductNode</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getAlternativeProductRibosomalFrameshiftingNode()">getAlternativeProductRibosomalFrameshiftingNode</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>com.ohnosequences.bio4j.blueprints.model.nodes.AlternativeProductNode</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getAlternativeProductSplicingNode()">getAlternativeProductSplicingNode</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>com.ohnosequences.bio4j.blueprints.model.nodes.citation.ArticleNode</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getArticleByDoiId(java.lang.String)">getArticleByDoiId</a></strong>(java.lang.String&nbsp;articleDoiId)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>com.ohnosequences.bio4j.blueprints.model.nodes.citation.ArticleNode</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getArticleByMedlineId(java.lang.String)">getArticleByMedlineId</a></strong>(java.lang.String&nbsp;articleMedlineId)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>com.ohnosequences.bio4j.blueprints.model.nodes.citation.ArticleNode</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getArticleByPubmedId(java.lang.String)">getArticleByPubmedId</a></strong>(java.lang.String&nbsp;articlePubmedId)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.citation.ArticleNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getArticlesByTitle(java.lang.String)">getArticlesByTitle</a></strong>(java.lang.String&nbsp;articleTitle)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>com.ohnosequences.bio4j.blueprints.model.nodes.GoTermNode</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getBiologicalProcessGoTerm()">getBiologicalProcessGoTerm</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.citation.BookNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getBooksByName(java.lang.String)">getBooksByName</a></strong>(java.lang.String&nbsp;bookName)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>com.ohnosequences.bio4j.blueprints.model.nodes.GoTermNode</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getCellularComponentGoTerm()">getCellularComponentGoTerm</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>com.ohnosequences.bio4j.blueprints.model.nodes.CityNode</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getCityNodeByName(java.lang.String)">getCityNodeByName</a></strong>(java.lang.String&nbsp;cityName)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>com.ohnosequences.bio4j.blueprints.model.nodes.CommentTypeNode</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getCommentTypeByName(java.lang.String)">getCommentTypeByName</a></strong>(java.lang.String&nbsp;commentTypeName)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>com.ohnosequences.bio4j.blueprints.model.nodes.ConsortiumNode</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getConsortiumByName(java.lang.String)">getConsortiumByName</a></strong>(java.lang.String&nbsp;consortiumName)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>com.ohnosequences.bio4j.blueprints.model.nodes.CountryNode</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getCountryNodeByName(java.lang.String)">getCountryNodeByName</a></strong>(java.lang.String&nbsp;countryName)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>com.ohnosequences.bio4j.blueprints.model.nodes.DatasetNode</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getDatasetByName(java.lang.String)">getDatasetByName</a></strong>(java.lang.String&nbsp;name)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>com.ohnosequences.bio4j.blueprints.model.nodes.citation.DBNode</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getDBByName(java.lang.String)">getDBByName</a></strong>(java.lang.String&nbsp;dbName)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>com.ohnosequences.bio4j.blueprints.model.nodes.EnzymeNode</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getEnzymeById(java.lang.String)">getEnzymeById</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>com.ohnosequences.bio4j.blueprints.model.nodes.FeatureTypeNode</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getFeatureTypeByName(java.lang.String)">getFeatureTypeByName</a></strong>(java.lang.String&nbsp;featureTypeName)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>com.ohnosequences.bio4j.blueprints.model.nodes.refseq.GenomeElementNode</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getGenomeElementByVersion(java.lang.String)">getGenomeElementByVersion</a></strong>(java.lang.String&nbsp;version)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>com.ohnosequences.bio4j.blueprints.model.nodes.GoTermNode</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getGoTermById(java.lang.String)">getGoTermById</a></strong>(java.lang.String&nbsp;goId)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>com.ohnosequences.bio4j.blueprints.model.nodes.InstituteNode</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getInstituteByName(java.lang.String)">getInstituteByName</a></strong>(java.lang.String&nbsp;instituteName)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>com.ohnosequences.bio4j.blueprints.model.nodes.InterproNode</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getInterproById(java.lang.String)">getInterproById</a></strong>(java.lang.String&nbsp;interproId)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>com.ohnosequences.bio4j.blueprints.model.nodes.IsoformNode</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getIsoformById(java.lang.String)">getIsoformById</a></strong>(java.lang.String&nbsp;isoformId)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>com.ohnosequences.bio4j.blueprints.model.nodes.citation.JournalNode</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getJournalByName(java.lang.String)">getJournalByName</a></strong>(java.lang.String&nbsp;journalName)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>com.ohnosequences.bio4j.blueprints.model.nodes.KeywordNode</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getKeywordById(java.lang.String)">getKeywordById</a></strong>(java.lang.String&nbsp;keywordId)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>com.ohnosequences.bio4j.blueprints.model.nodes.KeywordNode</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getKeywordByName(java.lang.String)">getKeywordByName</a></strong>(java.lang.String&nbsp;keywordName)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>com.ohnosequences.bio4j.model.nodes.GoTerm</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getMolecularFunctionGoTerm()">getMolecularFunctionGoTerm</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../com/ohnosequences/bio4j/titan/model/NCBITaxonNode.html" title="class in com.ohnosequences.bio4j.titan.model">NCBITaxonNode</a></code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getNCBITaxonByGiId(java.lang.String)">getNCBITaxonByGiId</a></strong>(java.lang.String&nbsp;giId)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code><a href="../../../../../../com/ohnosequences/bio4j/titan/model/NCBITaxonNode.html" title="class in com.ohnosequences.bio4j.titan.model">NCBITaxonNode</a></code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getNCBITaxonByTaxId(java.lang.String)">getNCBITaxonByTaxId</a></strong>(java.lang.String&nbsp;taxId)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.citation.OnlineArticleNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getOnlineArticlesByTitle(java.lang.String)">getOnlineArticlesByTitle</a></strong>(java.lang.String&nbsp;onlineArticleTitle)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>com.ohnosequences.bio4j.blueprints.model.nodes.citation.OnlineJournalNode</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getOnlineJournalByName(java.lang.String)">getOnlineJournalByName</a></strong>(java.lang.String&nbsp;onlineJournalName)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>com.ohnosequences.bio4j.blueprints.model.nodes.OrganismNode</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getOrganismByNCBITaxonomyId(java.lang.String)">getOrganismByNCBITaxonomyId</a></strong>(java.lang.String&nbsp;ncbiTaxonomyId)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>com.ohnosequences.bio4j.blueprints.model.nodes.OrganismNode</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getOrganismByScientificName(java.lang.String)">getOrganismByScientificName</a></strong>(java.lang.String&nbsp;scientificName)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>com.ohnosequences.bio4j.blueprints.model.nodes.citation.PatentNode</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getPatentByNumber(java.lang.String)">getPatentByNumber</a></strong>(java.lang.String&nbsp;patentNumber)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.PersonNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getPersonByName(java.lang.String)">getPersonByName</a></strong>(java.lang.String&nbsp;personName)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>com.ohnosequences.bio4j.blueprints.model.nodes.PfamNode</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getPfamById(java.lang.String)">getPfamById</a></strong>(java.lang.String&nbsp;pfamId)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinByAccession(java.lang.String)">getProteinByAccession</a></strong>(java.lang.String&nbsp;proteinAccession)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinByAllergomeReference(java.lang.String)">getProteinByAllergomeReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinByArrayExpressReference(java.lang.String)">getProteinByArrayExpressReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinByEmblReference(java.lang.String)">getProteinByEmblReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinByEnsemblPlantsRef(java.lang.String)">getProteinByEnsemblPlantsRef</a></strong>(java.lang.String&nbsp;ensemblPlantsRef)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinByFullName(java.lang.String)">getProteinByFullName</a></strong>(java.lang.String&nbsp;proteinFullName)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinByGeneNames(java.lang.String)">getProteinByGeneNames</a></strong>(java.lang.String&nbsp;proteinGeneName)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinByKEGGReference(java.lang.String)">getProteinByKEGGReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinByPIRReference(java.lang.String)">getProteinByPIRReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinByRefSeqReference(java.lang.String)">getProteinByRefSeqReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinByUniGeneReference(java.lang.String)">getProteinByUniGeneReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>private java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinReference(java.lang.String, java.lang.String)">getProteinReference</a></strong>(java.lang.String&nbsp;referenceName,
                   java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByArachnoServerReference(java.lang.String)">getProteinsByArachnoServerReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByBGEEReference(java.lang.String)">getProteinsByBGEEReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByBindingDBReference(java.lang.String)">getProteinsByBindingDBReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByBioCycReference(java.lang.String)">getProteinsByBioCycReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByBrendaReference(java.lang.String)">getProteinsByBrendaReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByCazyReference(java.lang.String)">getProteinsByCazyReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByCGDReference(java.lang.String)">getProteinsByCGDReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByCHEmblReference(java.lang.String)">getProteinsByCHEmblReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByCleanexReference(java.lang.String)">getProteinsByCleanexReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByCompluYeast2DPageReference(java.lang.String)">getProteinsByCompluYeast2DPageReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByConoServerReference(java.lang.String)">getProteinsByConoServerReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByCTDReference(java.lang.String)">getProteinsByCTDReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByCYGDReference(java.lang.String)">getProteinsByCYGDReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByDBSNPReference(java.lang.String)">getProteinsByDBSNPReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByDDBJReference(java.lang.String)">getProteinsByDDBJReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByDictyBaseReference(java.lang.String)">getProteinsByDictyBaseReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByDIPReference(java.lang.String)">getProteinsByDIPReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByDisprotReference(java.lang.String)">getProteinsByDisprotReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByDMDMReference(java.lang.String)">getProteinsByDMDMReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByDNASUReference(java.lang.String)">getProteinsByDNASUReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByDosacCobs2DPageReference(java.lang.String)">getProteinsByDosacCobs2DPageReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByEchoBaseReference(java.lang.String)">getProteinsByEchoBaseReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByEcoGeneReference(java.lang.String)">getProteinsByEcoGeneReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByEggNogReference(java.lang.String)">getProteinsByEggNogReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByEnsemblBacteriaReference(java.lang.String)">getProteinsByEnsemblBacteriaReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByEnsemblFungiReference(java.lang.String)">getProteinsByEnsemblFungiReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByEnsemblMetazoaReference(java.lang.String)">getProteinsByEnsemblMetazoaReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByEnsemblProtistsReference(java.lang.String)">getProteinsByEnsemblProtistsReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByEUHCVDBReference(java.lang.String)">getProteinsByEUHCVDBReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByEUPathDBReference(java.lang.String)">getProteinsByEUPathDBReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByEvolutionaryTraceReference(java.lang.String)">getProteinsByEvolutionaryTraceReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByFlyBaseReference(java.lang.String)">getProteinsByFlyBaseReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByGenAtlasReference(java.lang.String)">getProteinsByGenAtlasReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByGenBankReference(java.lang.String)">getProteinsByGenBankReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByGene3DReference(java.lang.String)">getProteinsByGene3DReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByGenecardsReference(java.lang.String)">getProteinsByGenecardsReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByGeneFarmReference(java.lang.String)">getProteinsByGeneFarmReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByGeneIDReference(java.lang.String)">getProteinsByGeneIDReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByGeneTreeReference(java.lang.String)">getProteinsByGeneTreeReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByGenevestigatorReference(java.lang.String)">getProteinsByGenevestigatorReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByGenolistReference(java.lang.String)">getProteinsByGenolistReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByGenomeReviewsReference(java.lang.String)">getProteinsByGenomeReviewsReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByGenomeRNAIReference(java.lang.String)">getProteinsByGenomeRNAIReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByGermOnlineReference(java.lang.String)">getProteinsByGermOnlineReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByGlycoSuiteDBReference(java.lang.String)">getProteinsByGlycoSuiteDBReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByGPCRDBReference(java.lang.String)">getProteinsByGPCRDBReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByGrameneReference(java.lang.String)">getProteinsByGrameneReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByHamapReference(java.lang.String)">getProteinsByHamapReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByHGNCReference(java.lang.String)">getProteinsByHGNCReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByHinvDBReference(java.lang.String)">getProteinsByHinvDBReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByHogenomReference(java.lang.String)">getProteinsByHogenomReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByHovergenReference(java.lang.String)">getProteinsByHovergenReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByHPAReference(java.lang.String)">getProteinsByHPAReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByHSSPReference(java.lang.String)">getProteinsByHSSPReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByHugeReference(java.lang.String)">getProteinsByHugeReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByIMGTReference(java.lang.String)">getProteinsByIMGTReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByInparanoidReference(java.lang.String)">getProteinsByInparanoidReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByIntactReference(java.lang.String)">getProteinsByIntactReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByIPIReference(java.lang.String)">getProteinsByIPIReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByKOReference(java.lang.String)">getProteinsByKOReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByLegioListReference(java.lang.String)">getProteinsByLegioListReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByLepromaReference(java.lang.String)">getProteinsByLepromaReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByMaizeGDBReference(java.lang.String)">getProteinsByMaizeGDBReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByMeropsReference(java.lang.String)">getProteinsByMeropsReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByMGIReference(java.lang.String)">getProteinsByMGIReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByMicadoReference(java.lang.String)">getProteinsByMicadoReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByMIMReference(java.lang.String)">getProteinsByMIMReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByMintReference(java.lang.String)">getProteinsByMintReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByModBaseReference(java.lang.String)">getProteinsByModBaseReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByMycoclapReference(java.lang.String)">getProteinsByMycoclapReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByNextBioReference(java.lang.String)">getProteinsByNextBioReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByNextProtReference(java.lang.String)">getProteinsByNextProtReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByOGPReference(java.lang.String)">getProteinsByOGPReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByOMAReference(java.lang.String)">getProteinsByOMAReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByOrphanetReference(java.lang.String)">getProteinsByOrphanetReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByOrthoDBReference(java.lang.String)">getProteinsByOrthoDBReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByPantherReference(java.lang.String)">getProteinsByPantherReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByPathwayReference(java.lang.String)">getProteinsByPathwayReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByPatricReference(java.lang.String)">getProteinsByPatricReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByPaxDBReference(java.lang.String)">getProteinsByPaxDBReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByPDBJReference(java.lang.String)">getProteinsByPDBJReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByPDBReference(java.lang.String)">getProteinsByPDBReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByPDBSumReference(java.lang.String)">getProteinsByPDBSumReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByPeptideAtlasReference(java.lang.String)">getProteinsByPeptideAtlasReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByPeroxiBaseReference(java.lang.String)">getProteinsByPeroxiBaseReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByPharmGKBReference(java.lang.String)">getProteinsByPharmGKBReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByPhosphositeReference(java.lang.String)">getProteinsByPhosphositeReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByPhosSiteReference(java.lang.String)">getProteinsByPhosSiteReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByPhylomeDBReference(java.lang.String)">getProteinsByPhylomeDBReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByPirsfReference(java.lang.String)">getProteinsByPirsfReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByPMapCutDBReference(java.lang.String)">getProteinsByPMapCutDBReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByPPTaseDBReference(java.lang.String)">getProteinsByPPTaseDBReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByPrideReference(java.lang.String)">getProteinsByPrideReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByPrintsReference(java.lang.String)">getProteinsByPrintsReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByProDomReference(java.lang.String)">getProteinsByProDomReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByPromexReference(java.lang.String)">getProteinsByPromexReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByProSiteReference(java.lang.String)">getProteinsByProSiteReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByProtClustDBReference(java.lang.String)">getProteinsByProtClustDBReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByProteinModelPortalReference(java.lang.String)">getProteinsByProteinModelPortalReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByProtonetReference(java.lang.String)">getProteinsByProtonetReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByPseudoCapReference(java.lang.String)">getProteinsByPseudoCapReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByRCSBPDBReference(java.lang.String)">getProteinsByRCSBPDBReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByReBaseReference(java.lang.String)">getProteinsByReBaseReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByReproductionReference(java.lang.String)">getProteinsByReproductionReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByRGDReference(java.lang.String)">getProteinsByRGDReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByRougeReference(java.lang.String)">getProteinsByRougeReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsBySBKBReference(java.lang.String)">getProteinsBySBKBReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsBySGDReference(java.lang.String)">getProteinsBySGDReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsBySmartReference(java.lang.String)">getProteinsBySmartReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsBySMRReference(java.lang.String)">getProteinsBySMRReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsBySourceReference(java.lang.String)">getProteinsBySourceReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByStringReference(java.lang.String)">getProteinsByStringReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsBySupfamReference(java.lang.String)">getProteinsBySupfamReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsBySwiss2DPageReference(java.lang.String)">getProteinsBySwiss2DPageReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByTairReference(java.lang.String)">getProteinsByTairReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByTCDBReference(java.lang.String)">getProteinsByTCDBReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByTigrFamsReference(java.lang.String)">getProteinsByTigrFamsReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByTuberculistReference(java.lang.String)">getProteinsByTuberculistReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByUCD2DPageReference(java.lang.String)">getProteinsByUCD2DPageReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByUCSCReference(java.lang.String)">getProteinsByUCSCReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByUniPathwayReference(java.lang.String)">getProteinsByUniPathwayReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByVectorBaseReference(java.lang.String)">getProteinsByVectorBaseReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByWorld2DPageReference(java.lang.String)">getProteinsByWorld2DPageReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByWormBaseReference(java.lang.String)">getProteinsByWormBaseReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByXenBaseReference(java.lang.String)">getProteinsByXenBaseReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getProteinsByZfinReference(java.lang.String)">getProteinsByZfinReference</a></strong>(java.lang.String&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>com.ohnosequences.bio4j.blueprints.model.nodes.citation.PublisherNode</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getPublisherByName(java.lang.String)">getPublisherByName</a></strong>(java.lang.String&nbsp;publisherName)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>com.ohnosequences.bio4j.blueprints.model.nodes.reactome.ReactomeTermNode</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getReactomeTermById(java.lang.String)">getReactomeTermById</a></strong>(java.lang.String&nbsp;reactomeTermId)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>com.ohnosequences.bio4j.blueprints.model.nodes.SequenceCautionNode</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getSequenceCautionErroneousGeneModelPredictionNode()">getSequenceCautionErroneousGeneModelPredictionNode</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>com.ohnosequences.bio4j.blueprints.model.nodes.SequenceCautionNode</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getSequenceCautionErroneousInitiationNode()">getSequenceCautionErroneousInitiationNode</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>com.ohnosequences.bio4j.blueprints.model.nodes.SequenceCautionNode</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getSequenceCautionErroneousTerminationNode()">getSequenceCautionErroneousTerminationNode</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>com.ohnosequences.bio4j.blueprints.model.nodes.SequenceCautionNode</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getSequenceCautionErroneousTranslationNode()">getSequenceCautionErroneousTranslationNode</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>com.ohnosequences.bio4j.blueprints.model.nodes.SequenceCautionNode</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getSequenceCautionFrameshiftNode()">getSequenceCautionFrameshiftNode</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>com.ohnosequences.bio4j.blueprints.model.nodes.SequenceCautionNode</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getSequenceCautionMiscellaneousDiscrepancyNode()">getSequenceCautionMiscellaneousDiscrepancyNode</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>com.ohnosequences.bio4j.blueprints.model.nodes.SubcellularLocationNode</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getSubcellularLocationByName(java.lang.String)">getSubcellularLocationByName</a></strong>(java.lang.String&nbsp;subcellularLocationName)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>com.ohnosequences.bio4j.blueprints.model.nodes.citation.SubmissionNode</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getSubmissionByTitle(java.lang.String)">getSubmissionByTitle</a></strong>(java.lang.String&nbsp;submissionTitle)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>com.ohnosequences.bio4j.model.nodes.Dataset</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getSwissProtDataset()">getSwissProtDataset</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>com.ohnosequences.bio4j.blueprints.model.nodes.TaxonNode</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getTaxonByName(java.lang.String)">getTaxonByName</a></strong>(java.lang.String&nbsp;taxonName)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.citation.ThesisNode&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getThesisByTitle(java.lang.String)">getThesisByTitle</a></strong>(java.lang.String&nbsp;thesisTitle)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>com.ohnosequences.bio4j.blueprints.model.nodes.DatasetNode</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html#getTremblDataset()">getTremblDataset</a></strong>()</code>&nbsp;</td>
</tr>
</table>
<ul class="blockList">
<li class="blockList"><a name="methods_inherited_from_class_java.lang.Object">
<!--   -->
</a>
<h3>Methods inherited from class&nbsp;java.lang.Object</h3>
<code>clone, equals, finalize, getClass, hashCode, notify, notifyAll, toString, wait, wait, wait</code></li>
</ul>
</li>
</ul>
</li>
</ul>
</div>
<div class="details">
<ul class="blockList">
<li class="blockList">
<!-- ============ FIELD DETAIL =========== -->
<ul class="blockList">
<li class="blockList"><a name="field_detail">
<!--   -->
</a>
<h3>Field Detail</h3>
<a name="manager">
<!--   -->
</a>
<ul class="blockListLast">
<li class="blockList">
<h4>manager</h4>
<pre>protected&nbsp;<a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/Bio4jManager.html" title="class in com.ohnosequences.bio4j.titan.model.util">Bio4jManager</a> manager</pre>
</li>
</ul>
</li>
</ul>
<!-- ========= CONSTRUCTOR DETAIL ======== -->
<ul class="blockList">
<li class="blockList"><a name="constructor_detail">
<!--   -->
</a>
<h3>Constructor Detail</h3>
<a name="NodeRetrieverTitan(com.ohnosequences.bio4j.titan.model.util.Bio4jManager)">
<!--   -->
</a>
<ul class="blockListLast">
<li class="blockList">
<h4>NodeRetrieverTitan</h4>
<pre>public&nbsp;NodeRetrieverTitan(<a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/Bio4jManager.html" title="class in com.ohnosequences.bio4j.titan.model.util">Bio4jManager</a>&nbsp;bio4jManager)</pre>
</li>
</ul>
</li>
</ul>
<!-- ============ METHOD DETAIL ========== -->
<ul class="blockList">
<li class="blockList"><a name="method_detail">
<!--   -->
</a>
<h3>Method Detail</h3>
<a name="getEnzymeById(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getEnzymeById</h4>
<pre>public&nbsp;com.ohnosequences.bio4j.blueprints.model.nodes.EnzymeNode&nbsp;getEnzymeById(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getEnzymeById</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getDatasetByName(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getDatasetByName</h4>
<pre>public&nbsp;com.ohnosequences.bio4j.blueprints.model.nodes.DatasetNode&nbsp;getDatasetByName(java.lang.String&nbsp;name)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getDatasetByName</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getSwissProtDataset()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getSwissProtDataset</h4>
<pre>public&nbsp;com.ohnosequences.bio4j.model.nodes.Dataset&nbsp;getSwissProtDataset()</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getSwissProtDataset</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getTremblDataset()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getTremblDataset</h4>
<pre>public&nbsp;com.ohnosequences.bio4j.blueprints.model.nodes.DatasetNode&nbsp;getTremblDataset()</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getTremblDataset</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getGenomeElementByVersion(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getGenomeElementByVersion</h4>
<pre>public&nbsp;com.ohnosequences.bio4j.blueprints.model.nodes.refseq.GenomeElementNode&nbsp;getGenomeElementByVersion(java.lang.String&nbsp;version)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getGenomeElementByVersion</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getGoTermById(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getGoTermById</h4>
<pre>public&nbsp;com.ohnosequences.bio4j.blueprints.model.nodes.GoTermNode&nbsp;getGoTermById(java.lang.String&nbsp;goId)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getGoTermById</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
<dt><span class="strong">Parameters:</span></dt><dd><code>goId</code> - </dd>
<dt><span class="strong">Returns:</span></dt><dd>GoTermNode with the id provided</dd></dl>
</li>
</ul>
<a name="getMolecularFunctionGoTerm()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getMolecularFunctionGoTerm</h4>
<pre>public&nbsp;com.ohnosequences.bio4j.model.nodes.GoTerm&nbsp;getMolecularFunctionGoTerm()</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getMolecularFunctionGoTerm</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getBiologicalProcessGoTerm()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getBiologicalProcessGoTerm</h4>
<pre>public&nbsp;com.ohnosequences.bio4j.blueprints.model.nodes.GoTermNode&nbsp;getBiologicalProcessGoTerm()</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getBiologicalProcessGoTerm</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getCellularComponentGoTerm()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getCellularComponentGoTerm</h4>
<pre>public&nbsp;com.ohnosequences.bio4j.blueprints.model.nodes.GoTermNode&nbsp;getCellularComponentGoTerm()</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getCellularComponentGoTerm</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinByAccession(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinByAccession</h4>
<pre>public&nbsp;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&nbsp;getProteinByAccession(java.lang.String&nbsp;proteinAccession)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinByAccession</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
<dt><span class="strong">Parameters:</span></dt><dd><code>proteinAccession</code> - </dd>
<dt><span class="strong">Returns:</span></dt><dd>ProteinNode with the accession provided</dd></dl>
</li>
</ul>
<a name="getProteinByEnsemblPlantsRef(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinByEnsemblPlantsRef</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinByEnsemblPlantsRef(java.lang.String&nbsp;ensemblPlantsRef)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinByEnsemblPlantsRef</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
<dt><span class="strong">Parameters:</span></dt><dd><code>ensemblPlantsRef</code> - </dd>
<dt><span class="strong">Returns:</span></dt><dd>ProteinNode with the Ensembl Plants reference provided</dd></dl>
</li>
</ul>
<a name="getProteinByFullName(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinByFullName</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinByFullName(java.lang.String&nbsp;proteinFullName)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinByFullName</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
<dt><span class="strong">Parameters:</span></dt><dd><code>proteinFullName</code> - </dd>
<dt><span class="strong">Returns:</span></dt><dd>List of proteins (if any) which match the full name provided</dd></dl>
</li>
</ul>
<a name="getProteinByGeneNames(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinByGeneNames</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinByGeneNames(java.lang.String&nbsp;proteinGeneName)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinByGeneNames</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
<dt><span class="strong">Parameters:</span></dt><dd><code>proteinGeneName</code> - </dd>
<dt><span class="strong">Returns:</span></dt><dd>List of proteins (if any) which match the gene name provided</dd></dl>
</li>
</ul>
<a name="getProteinByPIRReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinByPIRReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinByPIRReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinByPIRReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinByKEGGReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinByKEGGReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinByKEGGReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinByKEGGReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinByEmblReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinByEmblReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinByEmblReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinByEmblReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinByRefSeqReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinByRefSeqReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinByRefSeqReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinByRefSeqReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinByArrayExpressReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinByArrayExpressReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinByArrayExpressReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinByArrayExpressReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinByUniGeneReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinByUniGeneReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinByUniGeneReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinByUniGeneReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinByAllergomeReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinByAllergomeReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinByAllergomeReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinByAllergomeReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByArachnoServerReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByArachnoServerReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByArachnoServerReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByArachnoServerReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByBGEEReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByBGEEReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByBGEEReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByBGEEReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByBindingDBReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByBindingDBReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByBindingDBReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByBindingDBReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByBioCycReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByBioCycReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByBioCycReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByBioCycReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByBrendaReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByBrendaReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByBrendaReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByBrendaReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByCazyReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByCazyReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByCazyReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByCazyReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByCGDReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByCGDReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByCGDReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByCGDReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByCHEmblReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByCHEmblReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByCHEmblReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByCHEmblReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByCleanexReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByCleanexReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByCleanexReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByCleanexReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByCompluYeast2DPageReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByCompluYeast2DPageReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByCompluYeast2DPageReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByCompluYeast2DPageReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByConoServerReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByConoServerReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByConoServerReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByConoServerReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByCTDReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByCTDReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByCTDReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByCTDReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByCYGDReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByCYGDReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByCYGDReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByCYGDReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByDBSNPReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByDBSNPReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByDBSNPReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByDBSNPReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByDDBJReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByDDBJReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByDDBJReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByDDBJReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByDictyBaseReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByDictyBaseReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByDictyBaseReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByDictyBaseReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByDIPReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByDIPReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByDIPReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByDIPReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByDisprotReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByDisprotReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByDisprotReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByDisprotReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByDMDMReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByDMDMReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByDMDMReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByDMDMReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByDNASUReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByDNASUReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByDNASUReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByDNASUReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByDosacCobs2DPageReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByDosacCobs2DPageReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByDosacCobs2DPageReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByDosacCobs2DPageReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByEchoBaseReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByEchoBaseReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByEchoBaseReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByEchoBaseReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByEcoGeneReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByEcoGeneReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByEcoGeneReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByEcoGeneReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByEggNogReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByEggNogReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByEggNogReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByEggNogReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByEnsemblBacteriaReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByEnsemblBacteriaReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByEnsemblBacteriaReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByEnsemblBacteriaReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByEnsemblFungiReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByEnsemblFungiReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByEnsemblFungiReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByEnsemblFungiReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByEnsemblMetazoaReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByEnsemblMetazoaReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByEnsemblMetazoaReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByEnsemblMetazoaReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByEnsemblProtistsReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByEnsemblProtistsReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByEnsemblProtistsReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByEnsemblProtistsReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByEUHCVDBReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByEUHCVDBReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByEUHCVDBReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByEUHCVDBReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByEUPathDBReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByEUPathDBReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByEUPathDBReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByEUPathDBReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByEvolutionaryTraceReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByEvolutionaryTraceReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByEvolutionaryTraceReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByEvolutionaryTraceReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByFlyBaseReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByFlyBaseReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByFlyBaseReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByFlyBaseReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByGenAtlasReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByGenAtlasReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByGenAtlasReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByGenAtlasReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByGenBankReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByGenBankReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByGenBankReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByGenBankReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByGene3DReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByGene3DReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByGene3DReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByGene3DReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByGenecardsReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByGenecardsReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByGenecardsReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByGenecardsReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByGeneFarmReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByGeneFarmReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByGeneFarmReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByGeneFarmReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByGeneIDReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByGeneIDReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByGeneIDReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByGeneIDReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByGeneTreeReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByGeneTreeReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByGeneTreeReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByGeneTreeReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByGenevestigatorReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByGenevestigatorReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByGenevestigatorReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByGenevestigatorReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByGenolistReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByGenolistReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByGenolistReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByGenolistReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByGenomeReviewsReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByGenomeReviewsReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByGenomeReviewsReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByGenomeReviewsReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByGenomeRNAIReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByGenomeRNAIReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByGenomeRNAIReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByGenomeRNAIReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByGermOnlineReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByGermOnlineReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByGermOnlineReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByGermOnlineReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByGlycoSuiteDBReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByGlycoSuiteDBReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByGlycoSuiteDBReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByGlycoSuiteDBReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByGPCRDBReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByGPCRDBReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByGPCRDBReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByGPCRDBReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByGrameneReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByGrameneReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByGrameneReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByGrameneReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByHinvDBReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByHinvDBReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByHinvDBReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByHinvDBReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByHamapReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByHamapReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByHamapReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByHamapReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByHGNCReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByHGNCReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByHGNCReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByHGNCReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByHogenomReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByHogenomReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByHogenomReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByHogenomReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByHovergenReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByHovergenReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByHovergenReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByHovergenReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByHPAReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByHPAReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByHPAReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByHPAReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByHSSPReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByHSSPReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByHSSPReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByHSSPReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByHugeReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByHugeReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByHugeReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByHugeReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByIMGTReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByIMGTReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByIMGTReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByIMGTReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByInparanoidReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByInparanoidReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByInparanoidReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByInparanoidReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByIntactReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByIntactReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByIntactReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByIntactReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByIPIReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByIPIReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByIPIReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByIPIReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByKOReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByKOReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByKOReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByKOReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByLegioListReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByLegioListReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByLegioListReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByLegioListReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByLepromaReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByLepromaReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByLepromaReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByLepromaReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByMaizeGDBReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByMaizeGDBReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByMaizeGDBReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByMaizeGDBReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByMeropsReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByMeropsReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByMeropsReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByMeropsReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByMGIReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByMGIReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByMGIReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByMGIReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByMicadoReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByMicadoReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByMicadoReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByMicadoReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByMIMReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByMIMReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByMIMReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByMIMReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByMintReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByMintReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByMintReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByMintReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByModBaseReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByModBaseReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByModBaseReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByModBaseReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByMycoclapReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByMycoclapReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByMycoclapReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByMycoclapReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByNextBioReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByNextBioReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByNextBioReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByNextBioReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByNextProtReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByNextProtReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByNextProtReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByNextProtReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByOGPReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByOGPReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByOGPReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByOGPReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByOMAReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByOMAReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByOMAReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByOMAReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByOrphanetReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByOrphanetReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByOrphanetReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByOrphanetReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByOrthoDBReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByOrthoDBReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByOrthoDBReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByOrthoDBReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByPantherReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByPantherReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByPantherReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByPantherReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByPathwayReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByPathwayReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByPathwayReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByPathwayReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByPatricReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByPatricReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByPatricReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByPatricReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByPaxDBReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByPaxDBReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByPaxDBReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByPaxDBReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByPDBReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByPDBReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByPDBReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByPDBReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByPDBJReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByPDBJReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByPDBJReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByPDBJReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByPDBSumReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByPDBSumReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByPDBSumReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByPDBSumReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByPeptideAtlasReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByPeptideAtlasReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByPeptideAtlasReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByPeptideAtlasReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByPeroxiBaseReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByPeroxiBaseReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByPeroxiBaseReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByPeroxiBaseReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByPharmGKBReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByPharmGKBReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByPharmGKBReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByPharmGKBReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByPhosphositeReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByPhosphositeReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByPhosphositeReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByPhosphositeReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByPhosSiteReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByPhosSiteReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByPhosSiteReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByPhosSiteReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByPhylomeDBReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByPhylomeDBReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByPhylomeDBReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByPhylomeDBReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByPirsfReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByPirsfReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByPirsfReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByPirsfReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByPMapCutDBReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByPMapCutDBReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByPMapCutDBReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByPMapCutDBReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByPPTaseDBReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByPPTaseDBReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByPPTaseDBReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByPPTaseDBReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByPrideReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByPrideReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByPrideReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByPrideReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByPrintsReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByPrintsReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByPrintsReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByPrintsReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByProDomReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByProDomReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByProDomReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByProDomReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByPromexReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByPromexReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByPromexReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByPromexReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByProSiteReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByProSiteReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByProSiteReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByProSiteReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByProtClustDBReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByProtClustDBReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByProtClustDBReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByProtClustDBReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByProteinModelPortalReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByProteinModelPortalReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByProteinModelPortalReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByProteinModelPortalReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByProtonetReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByProtonetReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByProtonetReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByProtonetReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByPseudoCapReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByPseudoCapReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByPseudoCapReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByPseudoCapReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByRCSBPDBReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByRCSBPDBReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByRCSBPDBReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByRCSBPDBReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByReBaseReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByReBaseReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByReBaseReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByReBaseReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByReproductionReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByReproductionReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByReproductionReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByReproductionReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByRGDReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByRGDReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByRGDReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByRGDReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByRougeReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByRougeReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByRougeReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByRougeReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsBySBKBReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsBySBKBReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsBySBKBReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsBySBKBReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsBySGDReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsBySGDReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsBySGDReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsBySGDReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsBySmartReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsBySmartReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsBySmartReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsBySmartReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsBySMRReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsBySMRReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsBySMRReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsBySMRReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsBySourceReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsBySourceReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsBySourceReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsBySourceReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByStringReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByStringReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByStringReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByStringReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsBySupfamReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsBySupfamReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsBySupfamReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsBySupfamReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsBySwiss2DPageReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsBySwiss2DPageReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsBySwiss2DPageReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsBySwiss2DPageReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByTairReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByTairReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByTairReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByTairReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByTCDBReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByTCDBReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByTCDBReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByTCDBReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByTigrFamsReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByTigrFamsReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByTigrFamsReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByTigrFamsReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByTuberculistReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByTuberculistReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByTuberculistReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByTuberculistReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByUCD2DPageReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByUCD2DPageReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByUCD2DPageReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByUCD2DPageReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByUCSCReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByUCSCReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByUCSCReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByUCSCReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByUniPathwayReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByUniPathwayReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByUniPathwayReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByUniPathwayReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByVectorBaseReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByVectorBaseReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByVectorBaseReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByVectorBaseReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByWorld2DPageReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByWorld2DPageReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByWorld2DPageReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByWorld2DPageReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByWormBaseReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByWormBaseReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByWormBaseReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByWormBaseReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByXenBaseReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByXenBaseReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByXenBaseReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByXenBaseReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinsByZfinReference(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinsByZfinReference</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinsByZfinReference(java.lang.String&nbsp;id)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getProteinsByZfinReference</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getKeywordById(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getKeywordById</h4>
<pre>public&nbsp;com.ohnosequences.bio4j.blueprints.model.nodes.KeywordNode&nbsp;getKeywordById(java.lang.String&nbsp;keywordId)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getKeywordById</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
<dt><span class="strong">Parameters:</span></dt><dd><code>keywordId</code> - </dd>
<dt><span class="strong">Returns:</span></dt><dd>KeywordNode with the id provided</dd></dl>
</li>
</ul>
<a name="getKeywordByName(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getKeywordByName</h4>
<pre>public&nbsp;com.ohnosequences.bio4j.blueprints.model.nodes.KeywordNode&nbsp;getKeywordByName(java.lang.String&nbsp;keywordName)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getKeywordByName</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
<dt><span class="strong">Parameters:</span></dt><dd><code>keywordName</code> - </dd>
<dt><span class="strong">Returns:</span></dt><dd>KeywordNode with the id provided</dd></dl>
</li>
</ul>
<a name="getInterproById(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getInterproById</h4>
<pre>public&nbsp;com.ohnosequences.bio4j.blueprints.model.nodes.InterproNode&nbsp;getInterproById(java.lang.String&nbsp;interproId)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getInterproById</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
<dt><span class="strong">Parameters:</span></dt><dd><code>interproId</code> - </dd>
<dt><span class="strong">Returns:</span></dt><dd>InterproNode with the id provided</dd></dl>
</li>
</ul>
<a name="getPfamById(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getPfamById</h4>
<pre>public&nbsp;com.ohnosequences.bio4j.blueprints.model.nodes.PfamNode&nbsp;getPfamById(java.lang.String&nbsp;pfamId)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getPfamById</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
<dt><span class="strong">Parameters:</span></dt><dd><code>pfamId</code> - </dd>
<dt><span class="strong">Returns:</span></dt><dd>PfamNode with the id provided</dd></dl>
</li>
</ul>
<a name="getOrganismByScientificName(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getOrganismByScientificName</h4>
<pre>public&nbsp;com.ohnosequences.bio4j.blueprints.model.nodes.OrganismNode&nbsp;getOrganismByScientificName(java.lang.String&nbsp;scientificName)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getOrganismByScientificName</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
<dt><span class="strong">Parameters:</span></dt><dd><code>scientificName</code> - </dd>
<dt><span class="strong">Returns:</span></dt><dd>OrganismNode with the scientific name provided</dd></dl>
</li>
</ul>
<a name="getOrganismByNCBITaxonomyId(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getOrganismByNCBITaxonomyId</h4>
<pre>public&nbsp;com.ohnosequences.bio4j.blueprints.model.nodes.OrganismNode&nbsp;getOrganismByNCBITaxonomyId(java.lang.String&nbsp;ncbiTaxonomyId)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getOrganismByNCBITaxonomyId</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
<dt><span class="strong">Parameters:</span></dt><dd><code>ncbiTaxonomyId</code> - </dd>
<dt><span class="strong">Returns:</span></dt><dd>OrganismNode with the scientific name provided</dd></dl>
</li>
</ul>
<a name="getTaxonByName(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getTaxonByName</h4>
<pre>public&nbsp;com.ohnosequences.bio4j.blueprints.model.nodes.TaxonNode&nbsp;getTaxonByName(java.lang.String&nbsp;taxonName)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getTaxonByName</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
<dt><span class="strong">Parameters:</span></dt><dd><code>taxonName</code> - </dd>
<dt><span class="strong">Returns:</span></dt><dd>TaxonNode with the name provided</dd></dl>
</li>
</ul>
<a name="getNCBITaxonByTaxId(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getNCBITaxonByTaxId</h4>
<pre>public&nbsp;<a href="../../../../../../com/ohnosequences/bio4j/titan/model/NCBITaxonNode.html" title="class in com.ohnosequences.bio4j.titan.model">NCBITaxonNode</a>&nbsp;getNCBITaxonByTaxId(java.lang.String&nbsp;taxId)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getNCBITaxonByTaxId</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
<dt><span class="strong">Parameters:</span></dt><dd><code>taxId</code> - </dd>
<dt><span class="strong">Returns:</span></dt><dd>NCBITaxonNode with the tax id provided</dd></dl>
</li>
</ul>
<a name="getNCBITaxonByGiId(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getNCBITaxonByGiId</h4>
<pre>public&nbsp;<a href="../../../../../../com/ohnosequences/bio4j/titan/model/NCBITaxonNode.html" title="class in com.ohnosequences.bio4j.titan.model">NCBITaxonNode</a>&nbsp;getNCBITaxonByGiId(java.lang.String&nbsp;giId)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getNCBITaxonByGiId</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
<dt><span class="strong">Parameters:</span></dt><dd><code>giId</code> - </dd>
<dt><span class="strong">Returns:</span></dt><dd>NCBITaxonNode with the tax id provided</dd></dl>
</li>
</ul>
<a name="getIsoformById(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getIsoformById</h4>
<pre>public&nbsp;com.ohnosequences.bio4j.blueprints.model.nodes.IsoformNode&nbsp;getIsoformById(java.lang.String&nbsp;isoformId)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getIsoformById</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
<dt><span class="strong">Parameters:</span></dt><dd><code>isoformId</code> - </dd>
<dt><span class="strong">Returns:</span></dt><dd>IsoformNode with the id provided</dd></dl>
</li>
</ul>
<a name="getPersonByName(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getPersonByName</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.PersonNode&gt;&nbsp;getPersonByName(java.lang.String&nbsp;personName)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getPersonByName</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
<dt><span class="strong">Parameters:</span></dt><dd><code>personName</code> - </dd>
<dt><span class="strong">Returns:</span></dt><dd>PersonNode list with the name matching the value provided</dd></dl>
</li>
</ul>
<a name="getConsortiumByName(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getConsortiumByName</h4>
<pre>public&nbsp;com.ohnosequences.bio4j.blueprints.model.nodes.ConsortiumNode&nbsp;getConsortiumByName(java.lang.String&nbsp;consortiumName)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getConsortiumByName</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
<dt><span class="strong">Parameters:</span></dt><dd><code>consortiumName</code> - </dd>
<dt><span class="strong">Returns:</span></dt><dd>ConsortiumNode with the name provided</dd></dl>
</li>
</ul>
<a name="getInstituteByName(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getInstituteByName</h4>
<pre>public&nbsp;com.ohnosequences.bio4j.blueprints.model.nodes.InstituteNode&nbsp;getInstituteByName(java.lang.String&nbsp;instituteName)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getInstituteByName</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
<dt><span class="strong">Parameters:</span></dt><dd><code>instituteName</code> - </dd>
<dt><span class="strong">Returns:</span></dt><dd>InstituteNode with the name provided</dd></dl>
</li>
</ul>
<a name="getCountryNodeByName(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getCountryNodeByName</h4>
<pre>public&nbsp;com.ohnosequences.bio4j.blueprints.model.nodes.CountryNode&nbsp;getCountryNodeByName(java.lang.String&nbsp;countryName)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getCountryNodeByName</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
<dt><span class="strong">Parameters:</span></dt><dd><code>countryName</code> - </dd>
<dt><span class="strong">Returns:</span></dt><dd>CountryNode with the name provided</dd></dl>
</li>
</ul>
<a name="getCityNodeByName(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getCityNodeByName</h4>
<pre>public&nbsp;com.ohnosequences.bio4j.blueprints.model.nodes.CityNode&nbsp;getCityNodeByName(java.lang.String&nbsp;cityName)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getCityNodeByName</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
<dt><span class="strong">Parameters:</span></dt><dd><code>cityName</code> - </dd>
<dt><span class="strong">Returns:</span></dt><dd>CityNode with the name provided</dd></dl>
</li>
</ul>
<a name="getThesisByTitle(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getThesisByTitle</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.citation.ThesisNode&gt;&nbsp;getThesisByTitle(java.lang.String&nbsp;thesisTitle)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getThesisByTitle</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
<dt><span class="strong">Parameters:</span></dt><dd><code>thesisTitle</code> - </dd>
<dt><span class="strong">Returns:</span></dt><dd>ThesisNode list with the name matching the value provided</dd></dl>
</li>
</ul>
<a name="getPatentByNumber(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getPatentByNumber</h4>
<pre>public&nbsp;com.ohnosequences.bio4j.blueprints.model.nodes.citation.PatentNode&nbsp;getPatentByNumber(java.lang.String&nbsp;patentNumber)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getPatentByNumber</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
<dt><span class="strong">Parameters:</span></dt><dd><code>patentNumber</code> - </dd>
<dt><span class="strong">Returns:</span></dt><dd>PatentNode with the number provided</dd></dl>
</li>
</ul>
<a name="getBooksByName(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getBooksByName</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.citation.BookNode&gt;&nbsp;getBooksByName(java.lang.String&nbsp;bookName)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getBooksByName</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
<dt><span class="strong">Parameters:</span></dt><dd><code>bookName</code> - </dd>
<dt><span class="strong">Returns:</span></dt><dd>BookNode list with the name matching the value provided</dd></dl>
</li>
</ul>
<a name="getPublisherByName(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getPublisherByName</h4>
<pre>public&nbsp;com.ohnosequences.bio4j.blueprints.model.nodes.citation.PublisherNode&nbsp;getPublisherByName(java.lang.String&nbsp;publisherName)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getPublisherByName</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
<dt><span class="strong">Parameters:</span></dt><dd><code>publisherName</code> - </dd>
<dt><span class="strong">Returns:</span></dt><dd>PublisherNode with the name provided</dd></dl>
</li>
</ul>
<a name="getOnlineArticlesByTitle(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getOnlineArticlesByTitle</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.citation.OnlineArticleNode&gt;&nbsp;getOnlineArticlesByTitle(java.lang.String&nbsp;onlineArticleTitle)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getOnlineArticlesByTitle</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
<dt><span class="strong">Parameters:</span></dt><dd><code>onlineArticleTitle</code> - </dd>
<dt><span class="strong">Returns:</span></dt><dd>OnlineArticleNode list with the title matching the value provided</dd></dl>
</li>
</ul>
<a name="getOnlineJournalByName(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getOnlineJournalByName</h4>
<pre>public&nbsp;com.ohnosequences.bio4j.blueprints.model.nodes.citation.OnlineJournalNode&nbsp;getOnlineJournalByName(java.lang.String&nbsp;onlineJournalName)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getOnlineJournalByName</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
<dt><span class="strong">Parameters:</span></dt><dd><code>onlineJournalName</code> - </dd>
<dt><span class="strong">Returns:</span></dt><dd>OnlineJournalNode with the name provided</dd></dl>
</li>
</ul>
<a name="getArticlesByTitle(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getArticlesByTitle</h4>
<pre>public&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.citation.ArticleNode&gt;&nbsp;getArticlesByTitle(java.lang.String&nbsp;articleTitle)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getArticlesByTitle</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
<dt><span class="strong">Parameters:</span></dt><dd><code>articleTitle</code> - </dd>
<dt><span class="strong">Returns:</span></dt><dd>ArticleNode list with the title matching the value provided</dd></dl>
</li>
</ul>
<a name="getArticleByMedlineId(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getArticleByMedlineId</h4>
<pre>public&nbsp;com.ohnosequences.bio4j.blueprints.model.nodes.citation.ArticleNode&nbsp;getArticleByMedlineId(java.lang.String&nbsp;articleMedlineId)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getArticleByMedlineId</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
<dt><span class="strong">Parameters:</span></dt><dd><code>articleMedlineId</code> - </dd>
<dt><span class="strong">Returns:</span></dt><dd>ArticleNode with the medline id provided</dd></dl>
</li>
</ul>
<a name="getArticleByDoiId(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getArticleByDoiId</h4>
<pre>public&nbsp;com.ohnosequences.bio4j.blueprints.model.nodes.citation.ArticleNode&nbsp;getArticleByDoiId(java.lang.String&nbsp;articleDoiId)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getArticleByDoiId</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
<dt><span class="strong">Parameters:</span></dt><dd><code>articleDoiId</code> - </dd>
<dt><span class="strong">Returns:</span></dt><dd>ArticleNode with the DOI id provided</dd></dl>
</li>
</ul>
<a name="getArticleByPubmedId(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getArticleByPubmedId</h4>
<pre>public&nbsp;com.ohnosequences.bio4j.blueprints.model.nodes.citation.ArticleNode&nbsp;getArticleByPubmedId(java.lang.String&nbsp;articlePubmedId)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getArticleByPubmedId</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
<dt><span class="strong">Parameters:</span></dt><dd><code>articlePubmedId</code> - </dd>
<dt><span class="strong">Returns:</span></dt><dd>ArticleNode with the Pubmed id provided</dd></dl>
</li>
</ul>
<a name="getJournalByName(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getJournalByName</h4>
<pre>public&nbsp;com.ohnosequences.bio4j.blueprints.model.nodes.citation.JournalNode&nbsp;getJournalByName(java.lang.String&nbsp;journalName)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getJournalByName</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
<dt><span class="strong">Parameters:</span></dt><dd><code>journalName</code> - </dd>
<dt><span class="strong">Returns:</span></dt><dd>JournalNode with the name provided</dd></dl>
</li>
</ul>
<a name="getReactomeTermById(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getReactomeTermById</h4>
<pre>public&nbsp;com.ohnosequences.bio4j.blueprints.model.nodes.reactome.ReactomeTermNode&nbsp;getReactomeTermById(java.lang.String&nbsp;reactomeTermId)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getReactomeTermById</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
<dt><span class="strong">Parameters:</span></dt><dd><code>reactomeTermId</code> - </dd>
<dt><span class="strong">Returns:</span></dt><dd>ReactomeTermNode with the id provided</dd></dl>
</li>
</ul>
<a name="getFeatureTypeByName(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getFeatureTypeByName</h4>
<pre>public&nbsp;com.ohnosequences.bio4j.blueprints.model.nodes.FeatureTypeNode&nbsp;getFeatureTypeByName(java.lang.String&nbsp;featureTypeName)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getFeatureTypeByName</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
<dt><span class="strong">Parameters:</span></dt><dd><code>featureTypeName</code> - </dd>
<dt><span class="strong">Returns:</span></dt><dd></dd></dl>
</li>
</ul>
<a name="getCommentTypeByName(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getCommentTypeByName</h4>
<pre>public&nbsp;com.ohnosequences.bio4j.blueprints.model.nodes.CommentTypeNode&nbsp;getCommentTypeByName(java.lang.String&nbsp;commentTypeName)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getCommentTypeByName</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
<dt><span class="strong">Parameters:</span></dt><dd><code>commentTypeName</code> - </dd>
<dt><span class="strong">Returns:</span></dt><dd></dd></dl>
</li>
</ul>
<a name="getSubcellularLocationByName(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getSubcellularLocationByName</h4>
<pre>public&nbsp;com.ohnosequences.bio4j.blueprints.model.nodes.SubcellularLocationNode&nbsp;getSubcellularLocationByName(java.lang.String&nbsp;subcellularLocationName)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getSubcellularLocationByName</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
<dt><span class="strong">Parameters:</span></dt><dd><code>subcellularLocationName</code> - </dd>
<dt><span class="strong">Returns:</span></dt><dd></dd></dl>
</li>
</ul>
<a name="getAlternativeProductInitiationNode()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getAlternativeProductInitiationNode</h4>
<pre>public&nbsp;com.ohnosequences.bio4j.blueprints.model.nodes.AlternativeProductNode&nbsp;getAlternativeProductInitiationNode()</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getAlternativeProductInitiationNode</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getAlternativeProductPromoterNode()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getAlternativeProductPromoterNode</h4>
<pre>public&nbsp;com.ohnosequences.bio4j.blueprints.model.nodes.AlternativeProductNode&nbsp;getAlternativeProductPromoterNode()</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getAlternativeProductPromoterNode</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getAlternativeProductRibosomalFrameshiftingNode()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getAlternativeProductRibosomalFrameshiftingNode</h4>
<pre>public&nbsp;com.ohnosequences.bio4j.blueprints.model.nodes.AlternativeProductNode&nbsp;getAlternativeProductRibosomalFrameshiftingNode()</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getAlternativeProductRibosomalFrameshiftingNode</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getAlternativeProductSplicingNode()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getAlternativeProductSplicingNode</h4>
<pre>public&nbsp;com.ohnosequences.bio4j.blueprints.model.nodes.AlternativeProductNode&nbsp;getAlternativeProductSplicingNode()</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getAlternativeProductSplicingNode</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getSequenceCautionErroneousGeneModelPredictionNode()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getSequenceCautionErroneousGeneModelPredictionNode</h4>
<pre>public&nbsp;com.ohnosequences.bio4j.blueprints.model.nodes.SequenceCautionNode&nbsp;getSequenceCautionErroneousGeneModelPredictionNode()</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getSequenceCautionErroneousGeneModelPredictionNode</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getSequenceCautionErroneousInitiationNode()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getSequenceCautionErroneousInitiationNode</h4>
<pre>public&nbsp;com.ohnosequences.bio4j.blueprints.model.nodes.SequenceCautionNode&nbsp;getSequenceCautionErroneousInitiationNode()</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getSequenceCautionErroneousInitiationNode</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getSequenceCautionErroneousTranslationNode()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getSequenceCautionErroneousTranslationNode</h4>
<pre>public&nbsp;com.ohnosequences.bio4j.blueprints.model.nodes.SequenceCautionNode&nbsp;getSequenceCautionErroneousTranslationNode()</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getSequenceCautionErroneousTranslationNode</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getSequenceCautionErroneousTerminationNode()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getSequenceCautionErroneousTerminationNode</h4>
<pre>public&nbsp;com.ohnosequences.bio4j.blueprints.model.nodes.SequenceCautionNode&nbsp;getSequenceCautionErroneousTerminationNode()</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getSequenceCautionErroneousTerminationNode</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getSequenceCautionFrameshiftNode()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getSequenceCautionFrameshiftNode</h4>
<pre>public&nbsp;com.ohnosequences.bio4j.blueprints.model.nodes.SequenceCautionNode&nbsp;getSequenceCautionFrameshiftNode()</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getSequenceCautionFrameshiftNode</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getSequenceCautionMiscellaneousDiscrepancyNode()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getSequenceCautionMiscellaneousDiscrepancyNode</h4>
<pre>public&nbsp;com.ohnosequences.bio4j.blueprints.model.nodes.SequenceCautionNode&nbsp;getSequenceCautionMiscellaneousDiscrepancyNode()</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getSequenceCautionMiscellaneousDiscrepancyNode</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getSubmissionByTitle(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getSubmissionByTitle</h4>
<pre>public&nbsp;com.ohnosequences.bio4j.blueprints.model.nodes.citation.SubmissionNode&nbsp;getSubmissionByTitle(java.lang.String&nbsp;submissionTitle)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getSubmissionByTitle</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
<dt><span class="strong">Parameters:</span></dt><dd><code>submissionTitle</code> - </dd>
<dt><span class="strong">Returns:</span></dt><dd></dd></dl>
</li>
</ul>
<a name="getDBByName(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getDBByName</h4>
<pre>public&nbsp;com.ohnosequences.bio4j.blueprints.model.nodes.citation.DBNode&nbsp;getDBByName(java.lang.String&nbsp;dbName)</pre>
<dl>
<dt><strong>Specified by:</strong></dt>
<dd><code>getDBByName</code>&nbsp;in interface&nbsp;<code>com.ohnosequences.bio4j.model.util.NodeRetriever</code></dd>
</dl>
</li>
</ul>
<a name="getProteinReference(java.lang.String, java.lang.String)">
<!--   -->
</a>
<ul class="blockListLast">
<li class="blockList">
<h4>getProteinReference</h4>
<pre>private&nbsp;java.util.List&lt;com.ohnosequences.bio4j.blueprints.model.nodes.ProteinNode&gt;&nbsp;getProteinReference(java.lang.String&nbsp;referenceName,
                                                                                             java.lang.String&nbsp;id)</pre>
</li>
</ul>
</li>
</ul>
</li>
</ul>
</div>
</div>
<!-- ========= END OF CLASS DATA ========= -->
<!-- ======= START OF BOTTOM NAVBAR ====== -->
<div class="bottomNav"><a name="navbar_bottom">
<!--   -->
</a><a href="#skip-navbar_bottom" title="Skip navigation links"></a><a name="navbar_bottom_firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="../../../../../../overview-summary.html">Overview</a></li>
<li><a href="package-summary.html">Package</a></li>
<li class="navBarCell1Rev">Class</li>
<li><a href="class-use/NodeRetrieverTitan.html">Use</a></li>
<li><a href="package-tree.html">Tree</a></li>
<li><a href="../../../../../../deprecated-list.html">Deprecated</a></li>
<li><a href="../../../../../../index-files/index-1.html">Index</a></li>
<li><a href="../../../../../../help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList">
<li><a href="../../../../../../com/ohnosequences/bio4j/titan/model/util/Bio4jManager.html" title="class in com.ohnosequences.bio4j.titan.model.util"><span class="strong">Prev Class</span></a></li>
<li>Next Class</li>
</ul>
<ul class="navList">
<li><a href="../../../../../../index.html?com/ohnosequences/bio4j/titan/model/util/NodeRetrieverTitan.html" target="_top">Frames</a></li>
<li><a href="NodeRetrieverTitan.html" target="_top">No Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_bottom">
<li><a href="../../../../../../allclasses-noframe.html">All Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_bottom");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<div>
<ul class="subNavList">
<li>Summary:&nbsp;</li>
<li>Nested&nbsp;|&nbsp;</li>
<li><a href="#field_summary">Field</a>&nbsp;|&nbsp;</li>
<li><a href="#constructor_summary">Constr</a>&nbsp;|&nbsp;</li>
<li><a href="#method_summary">Method</a></li>
</ul>
<ul class="subNavList">
<li>Detail:&nbsp;</li>
<li><a href="#field_detail">Field</a>&nbsp;|&nbsp;</li>
<li><a href="#constructor_detail">Constr</a>&nbsp;|&nbsp;</li>
<li><a href="#method_detail">Method</a></li>
</ul>
</div>
<a name="skip-navbar_bottom">
<!--   -->
</a></div>
<!-- ======== END OF BOTTOM NAVBAR ======= -->
</body>
</html>
